docs/components/input.md

```diff
@@ -340,9 +340,51 @@ KInput allows you to listen to DOM events:
 </KComponent>
 ```
 
+## Expose
+
+### input
+
+KInput exposes HTML input element through `input` property for ease of performing manipulations.
+
+<div class="input-container">
+  <KInput 
+    ref="focusableKInput"
+    placeholder="Focusable input"
+    class="horizontal-spacing"
+  />
+  <KButton
+    size="large"
+    @click="setFocus"
+  >
+    Set focus
+  </KButton>
+</div>
+
+```vue
+<template>
+  <KInput 
+    ref="kInput"
```

Comment:
nit: can we rename the template ref here something other than `kInput` just to avoid some confusion?
```suggestion
    ref="myInputRef"
```